chaosazure/vmss/vmss_fetcher.py

```diff
@@ -0,0 +1,64 @@
+from azure.mgmt.compute import ComputeManagementClient
+from chaoslib import Configuration, Secrets
+
+from chaosazure import auth
+from chaosazure.rgraph.resource_graph import fetch_resources
+from chaosazure.vmss.constants import RES_TYPE_VMSS
+
+
+def fetch_vmss(query: str, configuration: Configuration, secrets: Secrets):
+    vmss = fetch_resources(query, RES_TYPE_VMSS, secrets, configuration)
+    results = __parse_vmss_result(vmss)
+    return results
+
+
+def fetch_vmss_instances(choice, configuration, secrets):
+    client = init_client(secrets, configuration)
+    pages = client.virtual_machine_scale_set_vms.list(
+        choice['resourceGroup'], choice['name'])
+    first_page = pages.advance_page()
+    vmss_instances = list(first_page)
+
+    while True:
+        try:
+            page = pages.advance_page()
+            vmss_instances.append(list(page))
```

Comment:
Just want to ensure this is what you want "a list of lists"?

Reply:
@Lawouach : Eagle eyes. Of course that is not what I wanted. Good catch - Fixed.